cpp/foxglove-websocket/include/foxglove/websocket/websocket_server.hpp

```diff
@@ -769,25 +851,36 @@ inline void Server<ServerConfiguration>::handleTextMessage(ConnHandle hdl, const
       unsubscribeParamsWithoutSubscriptions(hdl, paramNames);
     } break;
     case SUBSCRIBE_CONNECTION_GRAPH: {
```

Comment:
These are all getting so complicated that they should move into separate methods.